lab1b/lexic.txt

```diff
@@ -0,0 +1,32 @@
+Alphabet:
+    a. A-Z, a-z
+    b. decimal digits
+    c. underscore
+
+1. Lexic:
+
+    I. special symbols
+        a. operators: + - * / % < <= = == >= > !=
+        b. separators: () [] {} , ; space
```

Comment:
is `$` not a separator according to your syntax?

Reply:
I don't know if it counts as a separator